runtime/jvmti/jvmtiThread.c

```diff
@@ -1072,11 +1078,30 @@ resumeThread(J9VMThread * currentThread, jthread thread)
 	J9VMThread *targetThread = NULL;
 	jvmtiError rc = getVMThread(currentThread, thread, &targetThread, FALSE, TRUE);
 	if (rc == JVMTI_ERROR_NONE) {
-		if (targetThread->publicFlags & J9_PUBLIC_FLAGS_HALT_THREAD_JAVA_SUSPEND) {
-			clearHaltFlag(targetThread, J9_PUBLIC_FLAGS_HALT_THREAD_JAVA_SUSPEND);
-			Trc_JVMTI_threadResumed(targetThread);
-		} else {
-			rc = JVMTI_ERROR_THREAD_NOT_SUSPENDED;
+#if JAVA_SPEC_VERSION >= 19
+		BOOLEAN isVirtual = FALSE;
+		if (NULL != thread) {
+			isVirtual = IS_VIRTUAL_THREAD(currentThread, J9_JNI_UNWRAP_REFERENCE(thread));
+		}
+		if (isVirtual) {
+			/* targetThread is NULL only for an unmounted virtual threads. */
```

Comment:
grammar: please remove "an"

Reply:
None of the existing changes may stay. Will make sure it is fixed in the finalized changes if the comment stays.